catkit/hub/__init__.py

```diff
@@ -0,0 +1,415 @@
+import click
```

Comment:
This file seems to contain a lot of functions which are not necessarily called upon initialization? This will be difficult to read and difficult to document well in the future. I recommend moving these to their own module unless they need to be initialized upon calling catkit.hub

If your goal is to keep imports simple, you can use the strategy demonstrated for [gratoms initialization](https://github.com/SUNCAT-Center/CatKit/blob/master/catkit/__init__.py)

Reply:
Yes now we have all the functions for the cli in __init__.py. Guess that doesn't make sense - we should move it to a separate script.